superset-embedded-sdk/src/index.ts

```diff
@@ -154,6 +158,9 @@ export async function embedDashboard({
       });
 
       iframe.src = `${supersetDomain}/embedded/${id}${dashboardConfig}${filterConfigUrlParams}`;
+      if(iframeTitle) {
```

Comment:
I think given the default value set in the function header, this will always have a value, so no `if` required. Actually I think if someone actually wanted an empty string and pass `iframeTitle = ''` or `iframeTitle = null` in the function call, it would prevent them from overriding the `Embedded Dashboard` default :/